app/scripts/app.spec.js

```diff
@@ -344,8 +344,7 @@ angular.module('oncokbApp').run(
                 levelsDesc: {
                     '0': 'FDA-approved drug in this indication irrespective of gene/variant biomarker',
                     '1': 'FDA-recognized biomarker predictive of response to an FDA-approved drug in this indication',
```

Comment:
Please copy over the new level descriptions. I believe level 1 and 3A/B descriptions have been updated

Reply:
Updated